ClearOldData.php

```diff
@@ -20,6 +20,10 @@
 		":mail" => $cDataClearEmail
 	));
 	
+	if ($enableEmailConfirm == 1) {
```

Comment:
This code will cause a syntax error because you didn't add `$enableEmailConfirm` to the `global` directive on line 15.  Please remember to test your code to avoid issues like this.